catalogue/src/main/kotlin/com/tidal/sdk/catalogue/generated/infrastructure/CollectionFormats.kt

```diff
@@ -0,0 +1,56 @@
+package com.tidal.sdk.catalogue.generated.infrastructure
+
+class CollectionFormats {
```

Comment:
this looks like a great candidate for a sealed class but I assume this is also generated

Reply:
as you can see by the package name :-)